snap/snapcraft.yaml

```diff
@@ -132,6 +101,8 @@ apps:
 
 plugs:
   avahi-control: null
+  audio-playback: null
+  audio-record: null
```

Comment:
Do we actually need audio-record here?

Reply:
AFAIK, the control panel requires it for the mic volume control and the sound level bar.